### DISCUSSION TITLE
Why adding bias at the output of every GCN Layer?

### DISCUSSION THREAD

**Discussion by utkarsh0902311047** [pyg-team/pytorch_geometric#7982] [Q&A]
Hi,
1. Why add bias at the output of every GCN Layer?

I have one more doubt and want confirmation whether I am thinking correctly.
Does conv2 at the end of the model behave as a linear layer? Then cross_entropy will perform the log_softmax and Negative LogLikelihood Loss. Is this correct?

**Comment by rusty1s** [comment]
`GCN` would reduce to a `Linear` layer in case `edge_index` is empty, so it is naturally to also add a bias to it. You can customize this via `bias=False` when initializing the layer.
